p-process.yml

```diff
@@ -167,6 +167,9 @@ components:
     ProcessParams:
       type: object
       properties:
+        doDetectCan:
+          type: bool
+          description: "Enable the CAN (Card Access Number) detection when using scenarios with document location and MRZ reading, such as the MrzAndLocate scenario."
```

Comment:
Please make descriptions according to already defined style

Reply:
```suggestion
          description: "This parameter allows enabling the CAN (Card Access Number) detection when using scenarios with document location and MRZ reading, such as the MrzAndLocate scenario."
```